history/universe/marathon.json.mustache

```diff
@@ -56,9 +56,8 @@
 {{/security.kerberos.keytab}}
     "healthChecks": [
         {
-            "portIndex": 0,
-            "protocol": "MESOS_HTTP",
-            "path": "/",
+            "protocol": "COMMAND",
```

Comment:
What about setting the `SPARK_LOCAL_IP` to `0.0.0.0` when launching the history server?

Reply:
See https://jaceklaskowski.gitbooks.io/mastering-apache-spark/spark-history-server.html

Reply:
It looks like `0.0.0.0` is the default one but users might want to provide a specific binding (to private ip specifically). In case `SPARK_LOCAL_IP` is different from `0.0.0.0` or `127.0.0.1` `MESOS_HTTP` check will not work. Also, if a node has multiple network interfaces it doesn't look great to bind to all of them from the security perspective.